agent/api/task/service_connect.go

```diff
@@ -28,6 +28,10 @@ type ServiceConnectConfig struct {
 type RuntimeConfig struct {
 	// Host path for the administration socket
 	AdminSocketPath string `json:"adminSocketPath"`
+	// HTTP Path + Params to get statistical information
+	StatsRequest string `json:"statsRequest"`
```

Comment:
`TestStatsEngineWithServiceConnectMetrics` integ tests are failing since `StatsRequest` is added here, could you please update it so that `StatsRequest` is initialized [here](https://github.com/aws/amazon-ecs-agent/blob/feature/service-connect/agent/stats/engine_unix_integ_test.go#L94-L96)? Thank you!

Reply:
Thanks that was the missing piece. I updated hopefully appropriately.